### PR TITLE
feat(website): update ui-kit

### PR DESCRIPTION
**Pull request by kittybest**
# Description

Mainly just update the **ifm** configs, to change 
1) colors 
2) font-family 
3) font-size 
4) some arrangement.

## Related issue(s)

solve #871 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://github.com/privacy-scaling-explorations/maci/blob/dev/CONTRIBUTING.md) and [code of conduct](https://github.com/privacy-scaling-explorations/maci/blob/dev/CODE_OF_CONDUCT.md).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

**Comment by kittybest** [comment]
Setting it as draft now since I have some questions during building it, already asked the design team in dc channel.

File: website/src/pages/index.tsx [hunk 1]
```diff
@@ -2,16 +2,26 @@ import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
...
 import styles from "./index.module.css";
 
-const HomepageHeader = () => (
-  <header className={clsx("hero hero--primary", styles.heroBanner)}>
-    <div className="container">
-      <Logo height="50%" width="50%" />
+interface props {
+  tagline: string;
+  title: string;
+}
```
Comment (0xmad) [inline website/src/pages/index.tsx:12]:
```suggestion
interface HomepageHeaderProps {
  tagline: string;
  title: string;
}
```

File: website/src/css/custom.css [hunk 2]
```diff
@@ -4,27 +4,44 @@
  * work well for content-centric websites.
  */
 
+/* imported fonts */
+@import url("https://fonts.googleapis.com/css2?family=Share+Tech+Mono&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=DM+Mono&display=swap");
```
Comment (0xmad) [inline website/src/css/custom.css:9]: Let's download font locally.

File: website/src/components/HomepageFeatures/index.tsx [hunk 3]
```diff
@@ -46,10 +46,10 @@ const FeatureList: FeatureItem[] = [
   },
 ];
 
-const Feature = ({ title, Svg, description }: FeatureItem) => (
+const Feature = ({ title, img, description }: FeatureItem) => (
   <div className={clsx("col col--4")}>
     <div className="text--center">
-      <Svg className={styles.featureSvg} role="img" />
+      <img alt="none" className={styles.featureSvg} src={img} />
```
Comment (0xmad) [inline website/src/components/HomepageFeatures/index.tsx:52]: It's better to have alt for images and not ignore them

```suggestion
      <img alt={alt} className={styles.featureSvg} src={img} />
```

File: website/docusaurus.config.ts [hunk 4]
```diff
@@ -174,10 +172,10 @@ const config: Config = {
...
       copyright: `Copyright © ${new Date().getFullYear()} Privacy and Scaling Explorations`,
     },
-    prism: {
-      theme: prismThemes.github,
-      darkTheme: prismThemes.dracula,
-    },
+    // prism: {
+    //   theme: prismThemes.github,
+    //   darkTheme: prismThemes.dracula,
+    // },
```
Comment (0xmad) [inline website/docusaurus.config.ts:178]: Let's remove it

**Review by 0xmad — COMMENTED** [review]
@kittybest thanks, just few comments

File: website/src/css/custom.css (hunk 2 above)
Comment (kittybest) [reply website/src/css/custom.css:9]: Is there any reason for downloading fonts?

File: website/src/components/HomepageFeatures/index.tsx (hunk 3 above)
Comment (kittybest) [reply website/src/components/HomepageFeatures/index.tsx:52]: should be `title` right?

Comment (0xmad) [reply website/src/components/HomepageFeatures/index.tsx]: If it describes the image, yes.

File: website/src/css/custom.css [hunk 5]
```diff
@@ -4,27 +4,65 @@
  * work well for content-centric websites.
  */
 
+/* imported fonts */
+@font-face {
+  font-family: ShareTechMono;
+  src: url("../../static/fonts/Share_Tech_Mono/ShareTechMono-Regular.ttf");
```
Comment (0xmad) [inline website/src/css/custom.css:10]: Let's use woff2 format for both fonts. This one is supported for the most of the browsers now and no need to use old font format.

**Comment by samajammin** [comment]
Nice @kittybest!

A few issues I see on light mode:
Image 2024-01-17 at 12 12 42
- Nav links on hover turn white (so can't see text)
- Header text colors are white (so can't see text)
- Sidebar link for active route is white (so can't see text)
- Breadcrumb link is white (so hard to see text) - should probably be black text like the version badge

**Comment by samajammin** [comment]
> Nice @kittybest!
> 
> A few issues I see on light mode: 
> 
> * Nav links on hover turn white (so can't see text)
> * Header text colors are white (so can't see text)
> * Sidebar link for active route is white (so can't see text)
> * Breadcrumb link is white (so hard to see text) - should probably be black text like the version badge

HAH, my bad, this was all caused by a change I made 😆

File: website/static/img/pse-logo-round.png
Comment (samajammin) [inline website/static/img/pse-logo-round.png:1]: This file shouldn't have been deleted

File: website/src/css/custom.css [hunk 6]
```diff
@@ -4,27 +4,70 @@
...
+  --ifm-background-color: #ffffff;
+  --ifm-navbar-background-color: #ffffff;
+  --ifm-footer-background-color: #020203;
+  --ifm-footer-color: #ffffff;
+  --ifm-footer-link-color: #ffffff;
+  --ifm-footer-link-hover-color: #ffffff;
+  --ifm-link-color: #579bea;
+  --ifm-color-hero: #fcfcfc;
+  /* TODO: discuss if we want to use this or not */
+  /* --ifm-font-family-base: DMMono; */
```
Comment (samajammin) [inline website/src/css/custom.css:35]: Commented out font-family changes for now, pending discussion:
https://discord.com/channels/943612659163602974/1183821110844858462/1197273940075221163

**Review by samajammin — APPROVED** [review]

**Pull request merged**